docs/website/data/integrations.yaml

```diff
@@ -510,6 +510,23 @@ integrations:
       - styra
       - build.security
 
+  springsecurity-reactive:
+    title: Authorization for Java Spring Security Reactive
+    description: This integration provides a simple implementation of a ReactiveAuthorizationManager<ServerHttpRequest> for Spring Security Reactive that uses OPA for making API authorization decisions. In addition, the library provides out-of-the-box token-based JWT Authentication, allowing transparent RBAC implementations using JSON Web Tokens.
+    labels:
+      layer: network
+      category: application
+    software:
+      - javaspringsecurity
+    code:
+      - https://github.com/massenz/jwt-opa
+    tutorials:
+      - https://github.com/massenz/jwt-opa/blob/main/webapp-example/src/main/java/com/alertavert/opademo/JwtDemoApplication.java
+      - https://github.com/massenz/jwt-opa#web-server-demo-app
+    inventors:
```

Comment:
See the other examples from this file for how this should be done:
```
    inventors:
      - alertavert
```

Reply:
I was actually following [the instructions](https://github.com/open-policy-agent/opa/tree/master/docs#integration-index):
```
inventors: array of either

    - string (organization name)
    - object with fields
        - name: string
        - organization: string
```
(and I did look out for an example, L474 for `iptables` - although, admittedly, it seems to be most often used for the `speakers` fields)

Happy to change that, and (if you want me to) update the `integration-index` doc too.

Reply:
Thanks, had forgotten about that. You don't need to change the integration index, but if you can update the integration entry that'd be great 👍

Reply:
:+1: 
done.

Reply:
Thanks! Will merge as soon as build checks are green.